desktop/lib/app-instance/index.js

```diff
@@ -25,13 +25,11 @@ AppInstance.prototype.anotherInstanceStarted = function() {
 };
 
 AppInstance.prototype.isSingleInstance = function() {
-	let shouldQuit;
+	const gotTheLock = app.requestSingleInstanceLock()
 
-	shouldQuit = app.makeSingleInstance( this.anotherInstanceStarted.bind( this ) );
-
-	if ( shouldQuit ) {
+	if ( !gotTheLock ) {
```

Comment:
`gotTheLock` might seem a bit awkward. if we don't _need_ this variable and we don't use it beyond determining if this call succeeded, we can consider inlining it.

```js
AppInstance.prototype.isSingleInstance = function() {
  if (app.requestSingleInstanceLock()) {
    return true;
  }

  debug( 'App is already running, quitting' );
  app.quit();
  return false;
}
```

Reply:
Refactored in cbd6740